app/templates/main/record-view.html

```diff
@@ -0,0 +1,33 @@
+{% if file_type == "iiif" %}
+    <div class="govuk-tabs__panel" id="record-view">
+        <div class="record-view-header">
+            <a href="{{ url_for('main.download_record', record_id=record['file_id']) }}"
+               class="govuk-button govuk-button__download--record record-view-download-btn"
+               data-module="govuk-button">Download record</a>
+        </div>
+        <div class="universal-viewer" id="viewer">
+            <script nonce="{{ csp_nonce() }}"
+                    src="https://cdn.jsdelivr.net/npm/universalviewer@4.0.25/dist/umd/UV.min.js"></script>
```

Comment:
Would it be better to include a SRI hash in this resource and not rely on jsdelivr's auto optimisation?

```suggestion
            <script nonce="{{ csp_nonce() }}" src="https://cdn.jsdelivr.net/npm/universalviewer@4.0.25/dist/umd/UV.js" integrity="sha256-Fdp0oui1Jul9OK4Kuwm7OSbhzuuqBcgD5wi8A2tXTns=" crossorigin="anonymous"></script>
```

There is some advice on this: https://nationalarchives.github.io/engineering-handbook/third-party/cdn/

Reply:
I understand the benefit of adding out of interest but @ahosgood what do you mean by " not rely on jsdelivr's auto optimisation"?

Reply:
Universal Viewer contains the "main" file `UV.js`.

jsDelivr has a feature that automatically minifies non-minified resources. By requesting `UV.min.js`, you are asking for the version minified by the CDN rather than the original.

SRI doesn't work with files that have been optimised by jsDelivr.

<img width="1283" alt="image" src="https://github.com/user-attachments/assets/342e8352-480f-4182-a6c9-8456a962dd7a">

<img width="1278" alt="image" src="https://github.com/user-attachments/assets/75ea302c-ed26-4a7c-baaf-65e13d37bb6e">